lib/__test__/__snapshots__/query.test.js.snap

```diff
@@ -236,6 +236,76 @@ exports[`query getVariableFactors 2x 1`] = `
 ]
 `;
 
+exports[`query getVariableFactors 2x^2 * y 1`] = `
+[
+    {
+        "type": "Apply",
+        "op": "pow",
+        "args": [
+            {
+                "type": "Identifier",
+                "name": "x"
+            },
+            {
+                "value": "2",
+                "type": "Number"
+            }
+        ]
+    },
+    {
+        "type": "Identifier",
+        "name": "y"
+    }
+]
+`;
+
+exports[`query getVariableFactors 2x^2 * z^2 + y^3 1`] = `
```

Comment:
`y^3` is not a factor, it's a term.  If you want a function that finds all variable factors we should have a separate function for that.